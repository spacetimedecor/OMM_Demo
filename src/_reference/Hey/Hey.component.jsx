import * as React from "react";
import StyledHey from "./Hey.styled";

const Hey = ({
  children,
}) => (
  <StyledHey>
    Hey!
    {children}
  </StyledHey>
);

export default Hey;
