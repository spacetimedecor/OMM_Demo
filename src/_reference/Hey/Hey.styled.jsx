import styled from "styled-components";

const StyledHey = styled("div")`
  color: ${({ theme }) => theme.colors.primaryColor};
`;

export default StyledHey;
