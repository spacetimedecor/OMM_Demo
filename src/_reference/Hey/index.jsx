export { default } from "./Hey.component";
