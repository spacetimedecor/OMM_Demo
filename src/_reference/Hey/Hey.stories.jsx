import Hey from "./Hey.component";

import React from "react";

export default {
    title: 'Example/Hey',
    component: Hey,
    parameters: {
        // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
        layout: 'fullscreen',
    },
};

const Template = (args) => <Hey {...args} />;

export const Primary = Template.bind({});
Primary.args = {};
