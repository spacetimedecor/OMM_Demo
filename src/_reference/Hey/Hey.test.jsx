import * as React from "react";

import themed from "../../theme/themed";
import Hey from "./Hey.component";

describe("Hey", () => {
  it("should render correctly", () => {
    const component = themed(
      <Hey>
        Test
      </Hey>,
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
});
