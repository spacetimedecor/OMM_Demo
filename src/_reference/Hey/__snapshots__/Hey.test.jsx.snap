// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hey should render correctly 1`] = `
<div
  className="sc-bdvvtL bppPBt"
>
  Hey!
  Test
</div>
`;
